extensions/wikia/FounderEmails/events/FounderEmailsDaysPassedEvent.class.php

```diff
@@ -32,14 +32,14 @@ public function process( Array $events ) {
 			if ( empty( $emailController ) ) {
 				continue;
 			}
+			Wikia::initAsyncRequest( $wikiId );
```

Comment:
Fair enough, I like making things easier for our future selves. We spend enough time inadvertently making things more difficult, so being cautious for the sake of future us sounds good.